EDU-18598 - Feedback #5357 - Content improvement (#2792)
vtexdocs/dev-portal-content: docs/faststore/docs/getting-started/webops/moving-your-faststore-project-to-a-new-github-repository.mdx

Make sure the FastStore project you want to move uses [WebOps](https://developers.vtex.com/docs/guides/faststore/webops-dashboard).

### Have the required permissions

Make sure you have the following permissions before starting the migration:
- **VTEX Admin:** The `Owner` role in the VTEX account, which is the same role required to onboard a project to WebOps.
- **GitHub:** The `Admin` role in both the source and the target repositories.

### Have a GitHub organization

If you want to move your project to a new organization, make sure you have created a GitHub organization beforehand. Learn how to do so in the GitHub guide [Creating a new organization from scratch](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch).

### Have a GitHub repository

Make sure you have created the GitHub repository where you want to move your FastStore project. Learn how to do so in the GitHub guide [Creating a new repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository).

> >⚠️ The repository change is not available for **private GitHub instances**. If your project is connected to a private GitHub instance, the option won't be displayed.

Reviewer: ```suggestion
>⚠️ The repository change isn't available for **private GitHub instances**. The WebOps dashboard won't display the option if your project is connected to a private GitHub instance.
```
